doc/cephfs/troubleshooting.rst

```diff
@@ -375,9 +375,29 @@ and then reinstall it so that you have the latest version.
 Dynamic Debugging
 =================
 
-You can enable dynamic debug against the CephFS module.
+Dynamic debugging for CephFS kernel driver allows to enable or disable debug
+logging. The kernel driver logs are written to the kernel ring buffer and can
+be examined using ``dmesg(1)`` utility. Debug logging is disabled by default
+because enabling debug logging can result in system slowness a drop in I/O
```

Comment:
... system slowness and drop in...

Reply:
oof. Good catch. It'll be fixed in ninety seconds.